airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -282,6 +282,9 @@ def __init__(
         security_context: dict | None = None,
         container_security_context: dict | None = None,
         dnspolicy: str | None = None,
+        dns_config: dict | None = None,
```

Comment:
Kube has a type for this right?
```suggestion
        dns_config: V1PodDNSConfig | None = None,
```

Reply:
Good catch, I have pushed changes to reflect this.